BankClient/Bank.cs

```diff
@@ -2,165 +2,204 @@
 
 public class Bank
 {
-    private void menu()
+    private List<UserInformation> Users = new List<UserInformation>()
     {
-        Console.WriteLine("---------Menu---------");
-        Console.WriteLine("Enter 1 to add client\nEnter 2 to delete client\nEnter 3 to show list of users\nEnter 4 to search one user\nEnter 5 to search group of user\nEnter 6 to change user information");
-        Console.WriteLine("---------Menu ends---------");
-        Console.Write("Choose action: ");
+        new UserInformation("Dmitry", "Lapshin", "1234")
+            {ID = Guid.NewGuid()},
+        new UserInformation("Dima", "Lapshin", "1234")
+            {ID = Guid.NewGuid()}
```

Comment:
Move generating of identifier to constructor

Reply:
I did it